log/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package log provides common logging interface used either by baseplate.go users
+// or by other baseplate.go packages.
```

Comment:
Package log provides a wrapped zap logger interface for microservices to use, and also a simple Wrapper interface to be used by other baseplate.go packages.